docs/source/user-guide/nlp/supported_tasks.rst

```diff
@@ -0,0 +1,100 @@
+.. _supported_tasks:
+
+===================================
+Working with Labels and Predictions
```

Comment:
question - do you think the title here would be clearer for the user if it's:
Supported Task Types

Or something along those lines? (e.g. we can add in brackets "passing labels and predictions" or such). and same for the Tabular user guide, to call it "Supported Models"

Reply:
I want to convey the message that this is the place to look to understand what formats we support. Same in tabular, that's why "Supported Models" doesn't sound great to me for tabular - it sounds like "we support XGBoost".

What about "Supported Tasks and Label Formats" or something along these lines?